src/screens/reader/utils/handleNavigateChapter.ts

```diff
@@ -0,0 +1,67 @@
+import { useNovel } from '@hooks/reduxHooks';
+
+interface openChapterProps {
+  novel: {
+    novelUrl: string;
+    sourceId: string;
+    novelName: string;
+  };
+  chapter: {
+    chapterId: number;
+    chapterUrl: string;
+    novelId: string;
+    chapterName: string;
+    bookmark: boolean;
+  };
+  chapters: [
+    {
+      chapterId: number;
+      chapterName: string;
+      chapterUrl: string;
+    },
+  ];
+}
+interface selectFewerProps {
+  chapterId: number;
+  chapterName: string;
+  chapterUrl: string;
+}
+const selectFewerProps = (obj: selectFewerProps) => {
+  const { chapterId, chapterName, chapterUrl } = obj;
+  return { chapterId, chapterName, chapterUrl };
+};
+export function DopenChapter(
```

Comment:
Where's this function being used?

Reply:
Nowhere, I just used it for testing some things. This commit was just to see the reaction on the new function and wasn't finished yet. So I still haven't deleted all unnecessary code.

Reply:
Cool, you could mark the PR as Draft till it's ready.

Reply:
Sure, completely forgot about the existence of drafts.